docs/release.md

```diff
@@ -0,0 +1,92 @@
+# Releasing a new version of the Point of Sale Application
+
+All releases of this sample application are handled using CloudBuild triggers
+set up in the [`point-of-sale-ci`](https://console.cloud.google.com/cloud-build/triggers;region=global?project=point-of-sale-ci) GCP project. We use the custom written [releaser.py](/.github/releases/releaser.py) script for updating the configuration files with the correct
+versions during the release process. The release process includes **three stages**:
+1. Bumping the version in files _(pom.xml, package.json, Kubernetes manifests)_
+   to the required release version
+2. Publishing the release artifacts _(jar files, container images)_ to a central
+   repository and to Github
+3. Bumping the version in files _(pom.xml, package.json, Kubernetes manifests)_
+   to the next release version `SNAPSHOT`
+
+### Important resources regarding the release process:
+- [Versioning](#versioning) for this repository explained.
+- The [releaser.py](releaser.py) script that is used to bump the versions.
```

Comment:
this hyperlink will redirect to `main/docs/release`

Reply:
Fixed in 61895d48aba51ccd23bc192f94c470b5fcb5b64e